### DISCUSSION TITLE
提交代码fmt，出现多个文件变化

### DISCUSSION THREAD

**Discussion by panawala** [alibaba/ilogtail#297] [Help] [answered]
参考格式化Go代码：
https://ilogtail.gitbook.io/ilogtail-docs/v/pre-release/developer-guide/codestyle

出现多个文件变化
![image](https://user-images.githubusercontent.com/4111756/183550705-297d93ff-760d-44e0-9095-0145efdfac03.png)

变化如下：
![image](https://user-images.githubusercontent.com/4111756/183550777-3eeae6fa-5081-4224-a791-31f76f604366.png)

1. 上述变化是否正常？
2. 可否将fmt，加入到Makefile中，直接make fmt可以对代码进行格式化？

**Comment by yyuuttaaoo** [comment, marked as answer]
可能与go的版本有关，请单独发一个PR，只能包含格式变动，合入后以后就不会再变动了。
